eigen/LICENSE

```diff
@@ -0,0 +1 @@
+eigen/COPYING.README
```

Comment:
I think the component manager will not be able to determine the correct license in this case. It's probably better to copy the MPL license from eigen submodule here.

(cc @kumekay in case there are better options)